README.md

```diff
@@ -26,10 +26,10 @@ A Descope `Project ID` is required to initialize the SDK. Find it on the
 import "github.com/descope/go-sdk/descope/client"
 
 // Initialized after setting the DESCOPE_PROJECT_ID env var
-descopeClient := client.New()
+descopeClient, err := client.New()
```

Comment:
can we fix the rest of the examples?

![image](https://github.com/descope/go-sdk/assets/10514677/0e141ce3-d109-4452-8e25-1ba029004dfd)

Reply:
Done